### PR TITLE
[Model] reformat and update performance of grace

### PR DESCRIPTION
**Pull request by hengruizhang98**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
@mufeili

### CODE REVIEW

**Comment by hengruizhang98** [comment]
@dgl-bot run g4dn.4xlarge all dmlc/master

File: examples/pytorch/grace/model.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def sim(self, z1, z2):
 
     def get_loss(self, z1, z2):
         # calculate SimCLR loss
-
+
```
Comment (mufeili) [inline examples/pytorch/grace/model.py:75]: Remove it.

File: examples/pytorch/grace/README.md [hunk 2]
```diff
@@ -32,25 +32,47 @@ This example was implemented by [Hengrui Zhang](https://github.com/hengruizhang9
 --dataname         str     The graph dataset name.                Default is 'cora'.
 --gpu              int     GPU index.                             Default is 0.
 --split            int     Dataset spliting method.               Default is 'random'.
+--epochs           int     Number of training periods.            Default is 500.
+--lr               float   Learning rate.                         Default is 0.001.
+--wd               float   Weight decay.                          Default is 1e-5.
+--temp             float   Temperature.                           Default is 1.0.
+--act_fn           str     Activation function.                   Default is relu.001
```
Comment (mufeili) [inline examples/pytorch/grace/README.md:39]: "relu.001" -> "relu."

File: examples/pytorch/grace/main.py [hunk 3]
```diff
@@ -1,65 +1,81 @@
...
 
-    temp = config['tau']
-    epochs = config['num_epochs']
-    wd = config['weight_decay']
+    temp = args.temp
+    epochs = args.epochs
+    wd = args.wd
 
-    # Step 2: Prepare data =================================================================== #
+    # Step 2: Prepare data =================================================================== =#
```
Comment (mufeili) [inline examples/pytorch/grace/main.py:67]: Is "=" before "#" intended?

File: examples/pytorch/grace/README.md [hunk 4]
```diff
@@ -32,25 +32,47 @@ This example was implemented by [Hengrui Zhang](https://github.com/hengruizhang9
...
+--der1             float   Drop edge ratio 1.                     Default is 0.2. 
+--der2             float   Drop edge ratio 2.                     Default is 0.2. 
+--dfr1             float   Drop feature ratio 1.                  Default is 0.2. 
+--dfr2             float   Drop feature ratio 2.                  Default is 0.2. 
 ```
 
 ## How to run examples
 
-In the paper(as well as authors' repo), the training set and testing set are split randomly with 1:9 ratio. In order to fairly compare it with other models with the split they used, termed public split, in this repo we also provide experiment results using public split. To run the examples,
+In the paper(as well as authors' repo), the training set and testing set are split randomly with 1:9 ratio.  In order to fairly compare it with other methods with public split (20 training nodes each class), in this repo we also provide its results using public split (with fine-tuned hyper-parameters). To run the examples, follow the following instructions.
```
Comment (mufeili) [inline examples/pytorch/grace/README.md:51]: Extra space before "In"

Comment (mufeili) [inline examples/pytorch/grace/README.md:51]: "public split" -> "the public split"

File: examples/pytorch/grace/README.md [hunk 5]
```diff
@@ -32,25 +32,47 @@ This example was implemented by [Hengrui Zhang](https://github.com/hengruizhang9
...
+python main.py --dataname pubmed --epochs 1500 --lr 1e-3 --wd 1e-5 --hid_dim 256 --out_dim 256 --act_fn relu --der1 0.4 --der2 0.1 --dfr1 0.0 --dfr2 0.2 --temp 0.7
+
+# Pubmed with public split
+python main.py --dataname pubmed --split public --epochs 1500 --lr 1e-3 --wd 1e-5 --hid_dim 256 --out_dim 256 --act_fn relu --der1 0.4 --der2 0.1 --dfr1 0.0 --dfr2 0.2 --temp 0.7
+```
 
 ## 	Performance
 
-We use the same hyperparameter settings as provided by the author, you can check config.yaml for detailed hyper-parameters for each dataset.
+For random split, we use the hyper-parameters as stated in the paper. For public split,  we find the given hyper-parameters lead to poor performance, so we select the hyperparameters via  a small grid search.
```
Comment (mufeili) [inline examples/pytorch/grace/README.md:75]: wrong empty space before "a small grid search"

File: examples/pytorch/grace/main.py (hunk 3 above)
Comment (hengruizhang98) [reply examples/pytorch/grace/main.py:67]: it's a typo

**Review by mufeili — APPROVED** [review]
LGTM

**Pull request merged**